### PR TITLE
Hawkstrider attempt 3

### PR DESCRIPTION
**Pull request by LeeDarren**
Going to try manually breaking my files to get past the bot.

### CODE REVIEW

**Comment by LeeDarren** [comment]
bcc4cf6c149e34955d602328a5b11af337d913ac

**Comment by LeeDarren** [comment]
Please review this, @casey-collab

File: english/hawkstrider/startr/startr.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+The hawkstrider pauses for a moment then says:
+
+"Reaarrr. Urrrrr."
+
+A. [How ya doin fam?](../error/error.md)
+
+B. [Doral ana-diel?](../tier1/tier2/tier2.md)
```
Comment (casey-collab) [inline english/hawkstrider/startr/startr.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/hawkstrider/startr/startr.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+C. [Rearrrrr](../tier1/tier1.md)
```
Comment (casey-collab) [inline english/hawkstrider/startr/startr.md]: This link is also invalid.

File: english/hawkstrider/startu/startu.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+The hawkstrider pauses for a moment then says:
+
+"Urrrrrrrrr."
+
+A. [How ya doin fam?](../error/error.md)
+
+B. [Doral ana-diel?](../tier1/tier2/tier2.md)
```
Comment (casey-collab) [inline english/hawkstrider/startu/startu.md]: This link is also invalid.

File: english/hawkstrider/startu/startu.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+C. [Rearrrrr](../tier1/tier1.md)
```
Comment (casey-collab) [inline english/hawkstrider/startu/startu.md]: This link is also invalid.

File: english/hawkstrider/rude/rude.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+The hawkstrider leaps back, evidently startled.  It glares angrily at you for a moment,
```
Comment (casey-collab) [inline english/hawkstrider/rude/rude.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by LeeDarren** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Comment by LeeDarren** [comment]
For anyone who tries this, I am very sorry for providing a broken story.  I made a mistake and the bot won't let me update the rest of my commits.

**Pull request merged**